app/src/main/java/com/github/gogetters/letsgo/matchmaking/Matchmaking.kt

```diff
@@ -0,0 +1,37 @@
+package com.github.gogetters.letsgo.matchmaking
+
+import kotlin.math.pow
+
+class Matchmaking {
+    companion object {
+        // TODO if time is available, use glicko2 instead of elo
+        // https://lichess.org/page/rating-systems
+        // http://www.glicko.net/glicko/glicko2.pdf
+
+        private val eloChangeConstant = 30
+
+        private fun eloWinProbability(rating1: Double, rating2: Double): Double {
+            return 1.0 / (1.0 + 10.0.pow((rating1 - rating2) / 400.0))
```

Comment:
magic numbers all over the place

Reply:
I don't think there's anything I can name them, that's the formula I found